Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/viewmodel/SubmissionDoneNoConsentViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package de.rki.coronawarnapp.ui.submission.viewmodel
+
+import com.squareup.inject.assisted.AssistedInject
+import de.rki.coronawarnapp.util.ui.SingleLiveEvent
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModel
+import de.rki.coronawarnapp.util.viewmodel.SimpleCWAViewModelFactory
+
+class SubmissionDoneNoConsentViewModel @AssistedInject constructor() : CWAViewModel() {
+
+    val routeToScreen: SingleLiveEvent<SubmissionNavigationEvents> = SingleLiveEvent()
+
+    fun onBackPressed() {
+        routeToScreen.postValue(SubmissionNavigationEvents.NavigateToMainActivity)
+    }
+
+    fun onContinueWithSymptomRecordingPressed() {
+        routeToScreen.postValue(SubmissionNavigationEvents.NavigateToSymptomIntroduction)
+    }
+
+    fun onFlowFinished() {
```

Comment:
Hi @ritsam 
your code looks good overal :ok_hand:
If you want a cherry you could rename this a little more specific (external reader would not immediatly understand the reason for this event/fun by name).
Best.

Reply:
done